portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -669,6 +669,12 @@
 	<td>//td[contains(@class,'lfr-image-column')]</td>
 	<td></td>
 </tr>
+<!--MULTIPLE_SELECTION-->
+<tr>
+	<td>DELETE_OPTION_FROM_MULTIPLE_SELECTION</td>
+	<td>//input[contains(@value,'${key_value}')]//..//..//..//..//button[contains(@class,'close')]</td>
```

Comment:
@piresdm, can we try to create a path more specific?

I'm concerned about the maintenance of this path in the long term due to UI changes.

Reply:
@natocesarrego done!